src/test/java/api/loans/agetolost/ScheduledAgeToLostApiTest.java

```diff
@@ -0,0 +1,112 @@
+package api.loans.agetolost;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.matchers.ItemMatchers.isAgedToLost;
+import static api.support.matchers.ItemMatchers.isCheckedOut;
+import static api.support.matchers.ItemMatchers.isClaimedReturned;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.spring.SpringApiTest;
+import api.support.spring.clients.ScheduledJobClient;
+import lombok.val;
+
+public class ScheduledAgeToLostApiTest extends SpringApiTest {
```

Comment:
Given that these tests check the loan actions, is there a way to check that an item that is already aged to lost is not processed repeatedly?

Reply:
Yes, I've added a test case for this `shouldNotProcessAgedToLostItemSecondTime`